lib/rails/generators/neo4j/install/install_generator.rb

```diff
@@ -0,0 +1,27 @@
+require File.join(File.dirname(__FILE__), '..', '..', '..', '..', 'neo4j.rb')
+
+class Neo4j::Generators::InstallGenerator < Rails::Generators::Base
+  source_root File.expand_path("../templates", __FILE__)
+
+  desc "Install neo4j orm"
+
+  def copy_initializer
+
+    # ORM Config
+    if rails_5?
+      template 'neo4j_rails5.rb', 'config/initializers/neo4j.rb'
+    else
+      inject_into_file 'config/application.rb', File.read("../../templates/neo4j_rails4.rb"), :after => '[:password]'
+    end
+
+    # we are using JRuby - use Threads !
+    gsub_file "config/environments/production.rb", "# config.threadsafe!", "config.threadsafe!"
```

Comment:
From the first video of the seris about neo4j and rails, explain how to use the command: 
rails new myapp -m http://andreasronge.github.com/neo4j/rails.rb -O

That file http://neo4jrb.io/neo4j/neo4j.rb, you can see it here http://neo4jrb.io/neo4j/

And line 12 does that change in production.